### ISSUE TITLE
[google_sign_in]Cannot sign in on Android: `PlatformException: ra.b: 10: (sign_in_failed)` (sometimes, but not always!)

### ISSUE THREAD

**Issue by muccy-timeware** [flutter/flutter#136876]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/flutter/flutter/issues)
- [X] I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)

### Steps to reproduce

1. Integrate `google_sign_in` plugin in Android Flutter app
2. Launch Sign In procedure
3. Select a Google account

### Expected results

Get a Google Sign In Token

### Actual results

Despite having set SHA1 and SHA256 keys properly on Firebase Console, sometimes (*not* always) we receive a `PlatformException: ra.b: 10: (sign_in_failed)` as shown in attached video.
I underline that on the same phone with the same app, sometimes the problem does not appear.

We tried upgrading the plugin to latest version, but on some phones the problem persists.

The strange thing is that the same user, with same phone, with the same app installation sometimes sees the problem, sometimes not.

### Code sample

<details><summary>Code sample</summary>

```dart
final googleSignIn = GoogleSignIn(scopes: widget.scopes, clientId: widget.clientId);

    try {
      final account = await googleSignIn.signIn();
      if (account != null) {
        widget.onSuccess(account);
      } else {
        widget.onCancel?.call();
      }
    } catch (e) {
      widget.onFailure(e);
    } finally {
      setState(() {
        _isLoggingIn = false;
      });
    }
```

</details>

### Screenshots or Video

<details>
<summary>Screenshots / Video demonstration</summary>

https://github.com/flutter/flutter/assets/74927063/73b524eb-85c8-49ff-b009-0b3df2e53460

</details>

### Logs

<details><summary>Logs</summary>

N.A.

</details>

### Flutter Doctor output

<details><summary>Doctor output</summary>

```console
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.13.2, on macOS 14.0 23A344 darwin-arm64, locale it-IT)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.0)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.2)
[✓] VS Code (version 1.83.1)
[✓] Connected device (5 available)
[✓] Network resources

• No issues found!
```

</details>

**Comment by dam-ease** [comment]
Hi @muccy-timeware. Thanks for filing this. I see this doesn't reproduce at all times as I can't seem to reproduce on my end as well, can you share the devices you were able to reproduce this on and also, the full output of your `flutter run -v` to check the full logs?
Thank you!

**Comment by muccy-timeware** [comment]
Hi,
the issue happens on a wide range of devices (from a Moto E4 to a Pixel 4).

I've just been able to reproduce the error following these steps:

1. I installed the app from Play Store
2. I tried to Google Sign In with no success
3. I uninstalled the Play Store version
4. I launched the app with `flutter run -v`
5. I found the same error with different snackbar message
6. After 5 minutes the error disappeared

I attach you a piece of the complete verbose log (after app startup):

```
[        ] A Dart VM Service on Moto E 4 is available at: http://127.0.0.1:61163/7Hn0dwaNad8=/
[+1342 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Localization initialized[0m
[  +17 ms] The Flutter DevTools debugger and profiler on Moto E 4 is available at: http://127.0.0.1:9101?uri=http://127.0.0.1:61163/7Hn0dwaNad8=/
[ +120 ms] D/IubendaIAB(20140): Initializing Iubenda SDK
[   +1 ms] D/IubendaIAB(20140): Initialized v2.6.8 tcf2
[        ] D/IubendaIAB(20140): config: IubendaCMPConfig{gdprEnabled=true, forceConsent=false, googleAds=false, siteId='3254758', cookiePolicyId='15605809', cssResource=0, cssContent='null', cssFile=null, jsonResource=0, jsonContent='{"perPurposeConsent":true,"purposes":"1,4,5"}', jsonFile=null, cssUrl='null', dismissColor='-1', applyStyles=true, acceptIfDismissed=false, skipNoticeWhenOffline=true, preventDismissWhenLoaded=false, bannerPosition=CENTER, portraitWidth=-1, portraitHeight=-1, landscapeWidth=-1, landscapeHeight=-1, csVersion='current'}
[ +176 ms] I/Braze v23.3.0 .z4(20140): Setting Braze SDK disabled to: true
[        ] W/Braze v23.3.0 .Braze(20140): Stopping the SDK instance.
[        ] I/Braze v23.3.0 .Braze(20140): Shutting down all queued work on the Braze SDK
[        ] I/Braze v23.3.0 .BrazeCoroutineScope(20140): Cancelling children of BrazeCoroutineScope
[   +3 ms] I/Braze v23.3.0 .b5(20140): Cancelling children of SerialCoroutineScope
[        ] W/Braze v23.3.0 .Braze(20140): Disabling all network requests
[        ] I/Braze v23.3.0 .Braze(20140): Braze SDK outbound network requests are now disabled
[        ] W/Braze v23.3.0 .Braze(20140): SDK is disabled. Actions will not be performed on the SDK.
[+1136 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Start[0m
[   +1 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Init state[0m
[  +14 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Build[0m
[   +2 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Init Localization Delegate[0m
[   +3 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Init provider[0m
[+2197 ms] I/System.out(20140): Check isMmsSendPdu
[        ] I/System.out(20140): PDU read len:8192
[        ] I/System.out(20140): MMS PDU Type:31:139
[   +1 ms] I/System.out(20140): [OkHttp] sendRequest<<
[ +593 ms] I/System.out(20140): Wrong owner. Perhaps a late disconnect
[        ] I/System.out(20140): Wrong owner. Perhaps a late disconnect
[+2632 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Start[0m
[  +10 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Build[0m
[   +1 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Init Localization Delegate[0m
[   +3 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Init provider[0m
[+1780 ms] D/GraphicBuffer(20140): register, handle(0xaa135f40) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[ +609 ms] D/OpenGLRenderer(20140): ProgramCache.generateProgram: 34359738369
[+1001 ms] D/GraphicBuffer(20140): register, handle(0x84015940) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[ +992 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Load Localization Delegate[0m
[ +996 ms] D/libc-netbsd(20140): getaddrinfo: kfc-ws.imagonist.com get result from proxy gai_error = 0
[ +199 ms] D/libc-netbsd(20140): getaddrinfo: kfc-ws.imagonist.com get result from proxy gai_error = 0
[ +316 ms] D/GraphicBuffer(20140): register, handle(0x6a0cd900) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[+3413 ms] D/libc-netbsd(20140): getaddrinfo: kfc-ws.imagonist.com get result from proxy gai_error = 0
[  +12 ms] D/libc-netbsd(20140): getaddrinfo: kfc-ws.imagonist.com get result from proxy gai_error = 0
[ +934 ms] I/art     (20140): Do partial code cache collection, code=42KB, data=62KB
[   +1 ms] I/art     (20140): After code cache collection, code=41KB, data=61KB
[        ] I/art     (20140): Increasing code cache capacity to 256KB
[ +977 ms] D/libc-netbsd(20140): getaddrinfo: kfc-ws.imagonist.com get result from proxy gai_error = 0
[  +52 ms] D/libc-netbsd(20140): getaddrinfo: kfc-ws.imagonist.com get result from proxy gai_error = 0
[  +48 ms] I/Braze v23.3.0 .z4(20140): Setting Braze SDK disabled to: true
[        ] W/Braze v23.3.0 .Braze(20140): Stopping the SDK instance.
[        ] I/Braze v23.3.0 .Braze(20140): Shutting down all queued work on the Braze SDK
[        ] I/Braze v23.3.0 .BrazeCoroutineScope(20140): Cancelling children of BrazeCoroutineScope
[   +2 ms] I/Braze v23.3.0 .b5(20140): Cancelling children of SerialCoroutineScope
[        ] W/Braze v23.3.0 .Braze(20140): Disabling all network requests
[        ] I/Braze v23.3.0 .Braze(20140): Braze SDK outbound network requests are now disabled
[        ] W/Braze v23.3.0 .Braze(20140): SDK is disabled. Actions will not be performed on the SDK.
[  +10 ms] D/IubendaIAB(20140): askConsent
[        ] D/IubendaIAB(20140): shouldGetConsent: false
[+2298 ms] I/FA      (20140): Application backgrounded at: timestamp_millis: 1697722605316
[ +416 ms] W/FA      (20140): Cannot log screen view event when the app is in the background.
[ +686 ms] V/PhoneWindow(20140): DecorView setVisiblity: visibility = 0, Parent = ViewRoot{30312f2 com.kfc.it.mobileapp/com.kfc.it.mobileapp.MainActivity,ident = 0}, this = DecorView@22ad2d0[MainActivity]
[   +9 ms] V/InputMethodManager(20140): onWindowFocus: io.flutter.embedding.android.FlutterView{fbc300b VFE...... .F...... 0,0-720,1184 #2} softInputMode=16 first=false flags=#81810100
[ +138 ms] I/Braze v23.3.0 .Braze(20140): Braze SDK outbound network requests are now disabled
[   +4 ms] W/Braze v23.3.0 .Braze(20140): SDK is disabled. Actions will not be performed on the SDK.
[        ] I/art     (20140): Background partial concurrent mark sweep GC freed 22549(1696KB) AllocSpace objects, 12(240KB) LOS objects, 39% free, 5MB/9MB, paused 7.417ms total 102.023ms
[  +11 ms] I/Braze v23.3.0 .Braze(20140): Firebase Cloud Messaging found. Setting up Firebase Cloud Messaging.
[   +4 ms] I/Braze v23.3.0 .Braze(20140): Automatic ADM registration not enabled in configuration. Braze will not register for ADM.
[  +30 ms] I/Braze v23.3.0 .p1(20140): Geofences implicitly disabled via server configuration.
[+4230 ms] D/libc-netbsd(20140): getaddrinfo: media.imagonist.com get result from proxy gai_error = 0
[   +9 ms] D/libc-netbsd(20140): getaddrinfo: media.imagonist.com get result from proxy gai_error = 0
[        ] D/libc-netbsd(20140): getaddrinfo: media.imagonist.com get result from proxy gai_error = 0
[ +107 ms] D/libc-netbsd(20140): getaddrinfo: media.imagonist.com get result from proxy gai_error = 0
[        ] D/libc-netbsd(20140): getaddrinfo: media.imagonist.com get result from proxy gai_error = 0
[        ] D/libc-netbsd(20140): getaddrinfo: media.imagonist.com get result from proxy gai_error = 0
[+8641 ms] W/google_sign_in(20140): clientId is not supported on Android and is interpreted as serverClientId. Use serverClientId instead to suppress this warning.
[ +277 ms] W/Braze v23.3.0 .Braze(20140): SDK is disabled. Actions will not be performed on the SDK.
[  +24 ms] V/PhoneWindow(20140): DecorView setVisiblity: visibility = 4, Parent = null, this = DecorView@87628a3[]
[  +13 ms] D/WindowClient(20140): Add to mViews: DecorView@87628a3[SignInHubActivity], this = android.view.WindowManagerGlobal@7391f51
[   +5 ms] D/OpenGLRenderer(20140): CanvasContext() 0x66ea9bc0
[        ] D/ViewRootImpl[SignInHubActivity](20140): hardware acceleration is enabled, this = ViewRoot{a576bf6 com.kfc.it.mobileapp/com.google.android.gms.auth.api.signin.internal.SignInHubActivity,ident = 1}
[   +2 ms] V/PhoneWindow(20140): DecorView setVisiblity: visibility = 0, Parent = ViewRoot{a576bf6 com.kfc.it.mobileapp/com.google.android.gms.auth.api.signin.internal.SignInHubActivity,ident = 1}, this = DecorView@87628a3[SignInHubActivity]
[  +63 ms] D/Surface (20140): Surface::allocateBuffers(this=0x87e83800)
[        ] D/OpenGLRenderer(20140): CanvasContext() 0x66ea9bc0 initialize window=0x87e83800, title=com.kfc.it.mobileapp/com.google.android.gms.auth.api.signin.internal.SignInHubActivity
[        ] D/Surface (20140): Surface::connect(this=0x87e83800,api=1)
[        ] W/libEGL  (20140): [ANDROID_RECORDABLE] format: 1
[  +27 ms] D/mali_winsys(20140): EGLint new_window_surface(egl_winsys_display*, void*, EGLSurface, EGLConfig, egl_winsys_surface**, egl_color_buffer_format*, EGLBoolean) returns 0x3000
[  +31 ms] D/GraphicBuffer(20140): register, handle(0x65c520c0) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[  +14 ms] D/GraphicBuffer(20140): register, handle(0x65c52180) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[+3081 ms] I/FA      (20140): Application backgrounded at: timestamp_millis: 1697722623108
[+5770 ms] V/InputMethodManager(20140): onWindowFocus: null softInputMode=288 first=true flags=#1810100
[  +78 ms] W/Braze v23.3.0 .Braze(20140): SDK is disabled. Actions will not be performed on the SDK.
[  +14 ms] D/Surface (20140): Surface::disconnect(this=0x87e83800,api=1)
[   +5 ms] D/GraphicBuffer(20140): unregister, handle(0x65c520c0) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[        ] D/GraphicBuffer(20140): unregister, handle(0x65c52180) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[        ] D/Surface (20140): Surface::disconnect(this=0x87e83800,api=1)
[  +16 ms] D/WindowClient(20140): Remove from mViews: DecorView@87628a3[SignInHubActivity], this = android.view.WindowManagerGlobal@7391f51
[  +16 ms] V/PhoneWindow(20140): DecorView setVisiblity: visibility = 0, Parent = ViewRoot{30312f2 com.kfc.it.mobileapp/com.kfc.it.mobileapp.MainActivity,ident = 0}, this = DecorView@22ad2d0[MainActivity]
[   +2 ms] V/InputMethodManager(20140): onWindowFocus: io.flutter.embedding.android.FlutterView{fbc300b VFE...... .F...... 0,0-720,1184 #2} softInputMode=16 first=false flags=#81810100
[+11094 ms] W/google_sign_in(20140): clientId is not supported on Android and is interpreted as serverClientId. Use serverClientId instead to suppress this warning.
[  +81 ms] W/Braze v23.3.0 .Braze(20140): SDK is disabled. Actions will not be performed on the SDK.
[  +15 ms] V/PhoneWindow(20140): DecorView setVisiblity: visibility = 4, Parent = null, this = DecorView@99e0b24[]
[   +3 ms] D/WindowClient(20140): Add to mViews: DecorView@99e0b24[SignInHubActivity], this = android.view.WindowManagerGlobal@7391f51
[   +1 ms] D/OpenGLRenderer(20140): CanvasContext() 0x631ef840
[   +2 ms] D/ViewRootImpl[SignInHubActivity](20140): hardware acceleration is enabled, this = ViewRoot{60155c1 com.kfc.it.mobileapp/com.google.android.gms.auth.api.signin.internal.SignInHubActivity,ident = 2}
[   +6 ms] V/PhoneWindow(20140): DecorView setVisiblity: visibility = 0, Parent = ViewRoot{60155c1 com.kfc.it.mobileapp/com.google.android.gms.auth.api.signin.internal.SignInHubActivity,ident = 2}, this = DecorView@99e0b24[SignInHubActivity]
[  +46 ms] D/Surface (20140): Surface::allocateBuffers(this=0x65c34800)
[        ] D/OpenGLRenderer(20140): CanvasContext() 0x631ef840 initialize window=0x65c34800, title=com.kfc.it.mobileapp/com.google.android.gms.auth.api.signin.internal.SignInHubActivity
[        ] D/Surface (20140): Surface::connect(this=0x65c34800,api=1)
[   +1 ms] W/libEGL  (20140): [ANDROID_RECORDABLE] format: 1
[  +15 ms] D/mali_winsys(20140): EGLint new_window_surface(egl_winsys_display*, void*, EGLSurface, EGLConfig, egl_winsys_surface**, egl_color_buffer_format*, EGLBoolean) returns 0x3000
[  +10 ms] D/GraphicBuffer(20140): register, handle(0x65c52180) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[  +20 ms] D/GraphicBuffer(20140): register, handle(0x65c520c0) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[+2752 ms] I/FA      (20140): Application backgrounded at: timestamp_millis: 1697722643099
[+2403 ms] V/InputMethodManager(20140): onWindowFocus: null softInputMode=288 first=true flags=#1810100
[  +41 ms] W/Braze v23.3.0 .Braze(20140): SDK is disabled. Actions will not be performed on the SDK.
[   +3 ms] D/Surface (20140): Surface::disconnect(this=0x65c34800,api=1)
[   +1 ms] D/GraphicBuffer(20140): unregister, handle(0x65c52180) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[        ] D/GraphicBuffer(20140): unregister, handle(0x65c520c0) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[   +2 ms] D/Surface (20140): Surface::disconnect(this=0x65c34800,api=1)
[  +27 ms] D/WindowClient(20140): Remove from mViews: DecorView@99e0b24[SignInHubActivity], this = android.view.WindowManagerGlobal@7391f51
[  +14 ms] V/PhoneWindow(20140): DecorView setVisiblity: visibility = 0, Parent = ViewRoot{30312f2 com.kfc.it.mobileapp/com.kfc.it.mobileapp.MainActivity,ident = 0}, this = DecorView@22ad2d0[MainActivity]
[        ] V/InputMethodManager(20140): onWindowFocus: io.flutter.embedding.android.FlutterView{fbc300b VFE...... .F...... 0,0-720,1184 #2} softInputMode=16 first=false flags=#81810100
[+7734 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Start[0m
[   +1 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Init state[0m
[   +3 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Build[0m
[        ] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Init Localization Delegate[0m
[        ] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Init provider[0m
[ +298 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Start[0m
[   +1 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Build[0m
[   +7 ms] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Init Localization Delegate[0m
[        ] I/flutter (20140): [90m[🌎 Easy Localization] [DEBUG] Init provider[0m
[+30909 ms] W/google_sign_in(20140): clientId is not supported on Android and is interpreted as serverClientId. Use serverClientId instead to suppress this warning.
[ +221 ms] W/Braze v23.3.0 .Braze(20140): SDK is disabled. Actions will not be performed on the SDK.
[  +17 ms] V/PhoneWindow(20140): DecorView setVisiblity: visibility = 4, Parent = null, this = DecorView@fc03b68[]
[   +2 ms] D/WindowClient(20140): Add to mViews: DecorView@fc03b68[SignInHubActivity], this = android.view.WindowManagerGlobal@7391f51
[   +7 ms] D/OpenGLRenderer(20140): CanvasContext() 0x65bd5680
[        ] D/ViewRootImpl[SignInHubActivity](20140): hardware acceleration is enabled, this = ViewRoot{8834875 com.kfc.it.mobileapp/com.google.android.gms.auth.api.signin.internal.SignInHubActivity,ident = 3}
[   +4 ms] V/PhoneWindow(20140): DecorView setVisiblity: visibility = 0, Parent = ViewRoot{8834875 com.kfc.it.mobileapp/com.google.android.gms.auth.api.signin.internal.SignInHubActivity,ident = 3}, this = DecorView@fc03b68[SignInHubActivity]
[   +9 ms] I/art     (20140): Background sticky concurrent mark sweep GC freed 32647(2MB) AllocSpace objects, 27(544KB) LOS objects, 35% free, 5MB/9MB, paused 5.463ms total 45.037ms
[   +5 ms] D/OpenGLRenderer(20140): ~CanvasContext() 0x631ef840
[        ] D/OpenGLRenderer(20140): ~CanvasContext() 0x66ea9bc0
[   +9 ms] D/Surface (20140): Surface::allocateBuffers(this=0x63189000)
[        ] D/OpenGLRenderer(20140): CanvasContext() 0x65bd5680 initialize window=0x63189000, title=com.kfc.it.mobileapp/com.google.android.gms.auth.api.signin.internal.SignInHubActivity
[        ] D/Surface (20140): Surface::connect(this=0x63189000,api=1)
[   +2 ms] W/libEGL  (20140): [ANDROID_RECORDABLE] format: 1
[   +4 ms] D/mali_winsys(20140): EGLint new_window_surface(egl_winsys_display*, void*, EGLSurface, EGLConfig, egl_winsys_surface**, egl_color_buffer_format*, EGLBoolean) returns 0x3000
[  +33 ms] D/GraphicBuffer(20140): register, handle(0x65c520c0) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[  +15 ms] D/GraphicBuffer(20140): register, handle(0x65c52180) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[ +846 ms] D/SQLiteConnection(20140): execute took 558ms - succeeded, sql="COMMIT;"
[+2882 ms] I/FA      (20140): Application backgrounded at: timestamp_millis: 1697722688604
[+1873 ms] V/InputMethodManager(20140): onWindowFocus: null softInputMode=288 first=true flags=#1810100
[  +14 ms] W/Braze v23.3.0 .Braze(20140): SDK is disabled. Actions will not be performed on the SDK.
[   +9 ms] D/Surface (20140): Surface::disconnect(this=0x63189000,api=1)
[   +2 ms] D/GraphicBuffer(20140): unregister, handle(0x65c520c0) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[        ] D/GraphicBuffer(20140): unregister, handle(0x65c52180) (w:720 h:1184 s:720 f:0x1 u:0x000b00)
[        ] D/Surface (20140): Surface::disconnect(this=0x63189000,api=1)
[   +8 ms] D/WindowClient(20140): Remove from mViews: DecorView@fc03b68[SignInHubActivity], this = android.view.WindowManagerGlobal@7391f51
[  +12 ms] V/PhoneWindow(20140): DecorView setVisiblity: visibility = 0, Parent = ViewRoot{30312f2 com.kfc.it.mobileapp/com.kfc.it.mobileapp.MainActivity,ident = 0}, this = DecorView@22ad2d0[MainActivity]
[   +1 ms] W/IInputConnectionWrapper(20140): reportFullscreenMode on inexistent InputConnection
[        ] V/InputMethodManager(20140): onWindowFocus: io.flutter.embedding.android.FlutterView{fbc300b VFE...... .F...... 0,0-720,1184 #2} softInputMode=16 first=false flags=#81810100
[+27070 ms] Application finished.
```

I also attach you an updated video of the issue:

https://github.com/flutter/flutter/assets/74927063/bcb76753-4d8b-423e-a9d2-deab4c402a80


I also would like to add you that on Firebase console, we set SHA1/SHA256 both for local JKS and Play Console Signing key.

**Comment by dam-ease** [comment]
Hi @muccy-timeware. Thanks for responding and for the detailed explanation. I see similar issues currently open and some workarounds, please follow up on below similar issues below:

https://github.com/flutter/flutter/issues/75224
https://github.com/flutter/flutter/issues/33393

Closing this as a duplicate, if you disagree, please comment and I'll reopen this.
Thank you!
